project-base/storefront/components/Pages/Stores/StoresContent.tsx

```diff
@@ -37,63 +37,53 @@ export const StoresContent: FC<StoresContentProps> = ({ stores, breadcrumbs }) =
         return validMarkers;
     }, [mappedStores]);
 
-    const [activeMarkerId, setActiveMarkerId] = useState<string>();
+    const activeMarkerHandler = useCallback((markerId: string) => {
+        const newStoreIndex = mappedStores!.findIndex((store) => store.slug === markerId);
```

Comment:
This null assertion is unnecessary in my opinion. It does not help you in the code below, but can (potentially) cause issues.

Reply:
Ah, sure. It suppose to be Optional Chaining. Fixed.